keanu-project/src/test/resources/dotFiles/ConstantScalarIntNodeOutput.dot

```diff
@@ -1,7 +1,7 @@
 digraph BayesianNetwork {
 <91> -> <92> [label=right]
 <90> -> <92> [label=left]
-90[label="42"]
+90[label="42 (SomeLabel)"]
```

Comment:
Have you checked with users that this is the desired format? For example they might prefer `SomeLabel: 42` or `SomeLabel = 42`

Reply:
Discussed offline, this isn't a change introduced in this PR.